### PR TITLE
mdconfig: Print options when listing devices

### PR DESCRIPTION
**Pull request by ricardobranco777**
Currently, we have no way of seeing options set to md(4) devices.

Also fix usage string (missing `async` option).

Example:

`# mdconfig -a -f /root/backup.iso -o readonly -o verify`

Before:
```
# mdconfig -l -v
md0	vnode	 1536K	/root/backup.iso
```

After:
```
# mdconfig -l -v
md0	vnode	 1536K	/root/backup.iso	-	readonly,verify
```

Fixes https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=278624

### CODE REVIEW

File: sbin/mdconfig/mdconfig.c [hunk 1]
```diff
@@ -86,9 +87,9 @@ usage(void)
...
 "       mdconfig file\n");
 	fprintf(stderr, "\t\ttype = {malloc, vnode, swap}\n");
-	fprintf(stderr, "\t\toption = {cache, cluster, compress, force,\n");
-	fprintf(stderr, "\t\t          mustdealloc, readonly, reserve, ro,\n");
-	fprintf(stderr, "\t\t          verify}\n");
+	fprintf(stderr, "\t\toption = {async, cache, cluster, compress,\n");
+	fprintf(stderr, "\t\t          force, mustdealloc, readonly, ro,\n");
+	fprintf(stderr, "\t\t          reserve, verify}\n");
 	fprintf(stderr, "\t\tsize = %%d (512 byte blocks), %%db (B),\n");
 	fprintf(stderr, "\t\t       %%dk (kB), %%dm (MB), %%dg (GB), \n");
```
Comment (bsdimp) [inline sbin/mdconfig/mdconfig.c:94]: Should delete this space after the , and before the \n

**Comment by bsdimp** [comment]
Otherwise, I like this.

File: sbin/mdconfig/mdconfig.c (hunk 1 above)
Comment (ricardobranco777) [reply sbin/mdconfig/mdconfig.c]: > Should delete this space after the , and before the \n

Fixed, thanks!

**Comment by bsdimp** [comment]
Thanks for this! Please close the PR now that this is in.

**Comment by asomers** [comment]
@ricardobranco777 what is the purpose of the `\0` separator?  Should that initially be `''`?  Currently, there is always an actual `\0` character printed at the beginning of the option list.

**Comment by ricardobranco777** [comment]
> @ricardobranco777 what is the purpose of the `\0` separator? Should that initially be `''`? Currently, there is always an actual `\0` character printed at the beginning of the option list.

You're right.  It must be ''.  My bad.  Will submit a PR tomorrow.

**Comment by bsdimp** [comment]
> > @ricardobranco777 what is the purpose of the `\0` separator? Should that initially be `''`? Currently, there is always an actual `\0` character printed at the beginning of the option list.
> 
> You're right. It must be `''`. My bad. Will submit a PR tomorrow.

Doh! Looks like I missed seeing that. Thanks for fixing.

**Pull request merged**